task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +524,38 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let result;
+
+    // Array of successful combinations
+    let successCombinArr = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  
+    // Loop to change voids to null and then apply the flat method
+    for(let i = 0; i < position.length; i++) {
+      position[i].length = 3;
+  
+      for(let j = 0; j < position[i].length; j++) {
+        if(!position[i][j]) {
+          position[i][j] = null;
+        }
+      }
+    }
+
+    let currArr = position.flat();
+  
+    successCombinArr.some((comb) => {
```

Comment:
return value of the **some** method is not used. 

You shouldn't use mtehods like everyt,some, map, filter etc. if you only need to iterate over the array use either loop or foreach method